webhook/runtime_webhooks.md

```diff
@@ -0,0 +1,51 @@
+
+For long-running (async) actions and guaranteed event handling you should consider using the [journaling approach](https://adobeio-codelabs-journaling-events-adobedocs.hlx.page/?src=/README.html) for consuming events.
+
+
+## Setting up Webhook Integration with Runtime Action
+
+The Runtime cli will let you create a runtime action and hook it up with an integration via Adobe Developer Console. Read [here](https://www.adobe.io/apis/experienceplatform/runtime/docs.html#!adobedocs/adobeio-runtime/master/getting-started/setup.md) to setup your cli with Runtime plugin which is required as pre-requisite.
+
+- Go to Developer Console and start creating an event registration
+- Add [Runtime](https://www.adobe.io/apis/experienceplatform/console/docs.html#!AdobeDocs/adobeio-console/master/services-enable-runtime.md) to your project, this will create the required auth and runtime namespace for you.
+- Go to `aio-cli` and create and deploy your user-action in your namespace. 
+- Come back to Developer Console and you will see the deployed runtime action in your namespace under the `User Defined Actions` section. Refresh the page if you don't see the action.
+- Now set up an event registration using the runtime action you just deployed. You would need to select the `Runtime Action` option in your `Configure event registration` page. Choose your runtime action from the dropdown and click `Save configured events`.
+- This will create an event registration with an event handler webhook pointing to your runtime action. 
+
+
+## Benefits of using Runtime Action as Webhook
+
+- [Built In Signature Verification](#built-in-signature-verification)
+- [Tracing Actions with Activation Ids](#tracing-actions-with-activation-ids)
+
+### Built In Signature Verification
+
+With integration between I/O Events and Adobe I/O Runtime, your runtime actions as webhooks are made secure by being provided with an OOTB signature verification implementation that's working under the hood. So, basically whatever runtime action you use to create an event registration the handler webhook created due to that will place a signature validator action along with your action as in a [sequence](#https://github.com/AdobeDocs/adobeio-runtime/blob/master/reference/sequences_compositions.md). I/O Events signs the event payload with client's secret signature and pass them in request header to webhook. Whatever requests will come to your handler webhook will only invoke your business logic runtime action when the validator action successfully verifies that the correct signature has been passed to invoke your webhook.
+
+### Tracing Actions with Activation Ids
+
+[Debug Tracing](#https://www.adobe.io/apis/experienceplatform/events/docs.html#!adobedocs/adobeio-events/master/support/tracing.md) is a pretty important tool on Developer Console for users who want to be informed whether their runtime action invocation is successful or not or what it responds.
```

Comment:
Again use relative path for the link here and see if it is getting rendered in git. This will validate the link that you are using.